### PR TITLE
Spotify playlist management:

### PR DESCRIPTION
**Pull request by qstorey**
- Improve playlist creation output.
- Add functionality to list playlists.

### CODE REVIEW

File: src/command.rs [hunk 1]
```diff
@@ -13,13 +13,33 @@ pub fn authenticate(spotify_client_id: &str, spotify_client_secret: &str) {
...
 pub fn create_playlist(spotify_client_id: &str, spotify_client_secret: &str, matches: &ArgMatches) {
     let name = matches.value_of("name").unwrap();
     let spotify_wrapper =
         spotify::SpotifyWrapper::new(&spotify_client_id, &spotify_client_secret).unwrap();
-    let playlist = spotify_wrapper.create_playlist(&name).unwrap();
-    dbg!(&playlist);
+    let playlist = match spotify_wrapper.create_playlist(&name) {
+        Ok(p) => p,
+        Err(e) => {
+            eprintln!("{}", e);
```
Comment (hannesdejager) [inline src/command.rs:24]: For improved print debugging :-) you can use the debug format specifier i.e. `eprintln!("{:?}", e);`

Comment (hannesdejager) [reply src/command.rs:24]: OK wait you actually want the output to go to the user. Nevermind

**Comment by hannesdejager** [comment]
Looks neat :-) to me

File: src/command.rs (hunk 1 above)
Comment (qstorey) [reply src/command.rs:24]: Yeah, but I do want to come back to the error handling so I can improve output to the user.

**Pull request merged**